plugins/azure-devops/src/api/AzureDevOpsClient.ts

```diff
@@ -174,6 +179,7 @@ export class AzureDevOpsClient implements AzureDevOpsApi {
     if (options?.top) {
       queryString.append('top', options.top.toString());
     }
+    queryString.append('entityRef', entityRef);
```

Comment:
This is missing for the `getReadme` function lower down on about line 198, you'll get an error.

Need to add this: `queryString.append('entityRef', opts.entityRef);`

Reply:
Updated!
Thanks! I missed it during commit.

Reply:
No worries! Thanks for fixing.